scenarios/cicd/README.md

```diff
@@ -0,0 +1,70 @@
+# Scenario: cicd
+
+**Size:** Medium
```

Comment:
Size and difficulty TBD. I don't have enough knowledge of the other scenarios to judge